backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/UserPatchModel.java

```diff
@@ -9,8 +9,8 @@
 
 @Immutable
 @Schema(name = "UserUpdate")
```

Comment:
This should be updated to match the model name.

Reply:
fixed